main.go

```diff
@@ -36,7 +36,8 @@ var validParsers = []string{
 
 // GlobalOptions has all the top level CLI flags that honeytail supports
 type GlobalOptions struct {
-	APIHost string `hidden:"true" long:"api_host" description:"Host for the Honeycomb API" default:"https://api.honeycomb.io/"`
+	APIHost    string `hidden:"true" long:"api_host" description:"Host for the Honeycomb API" default:"https://api.honeycomb.io/"`
+	TailSample bool   `hidden:"true" long:"tail_sample" description:"When true, sample while tailing. When false, sample post-parser events"`
```

Comment:
-> "When true, drop 'sampled' lines before handing them to the parser. Should not be used for log types likely to have multi-line events (e.g. MySQL)"